File: Convertex2/app/src/main/java/com/jain/convertex/MainActivity.java
package com.jain.convertex;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    public void btnClick(View view){
        EditText da = (EditText) findViewById(R.id.amount);
        String dollar = da.getText().toString();
        Double dollar1 = Double.parseDouble(dollar);
        Double dollar2 = 76.83 * dollar1;
        String toast = "=" + dollar2.toString();

        Toast.makeText(this, toast, Toast.LENGTH_SHORT).show();
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: GitHub/App-Dev/InstaUI/app/src/main/java/com/jain/instaui/MainActivity.kt
package com.jain.instaui

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
         menuInflater.inflate(R.menu.menu_bar,menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when(item.itemId) {
            R.id.miPost -> Toast.makeText(this,"Selected Post",Toast.LENGTH_SHORT).show()
            R.id.miStory -> Toast.makeText(this,"Selected Story",Toast.LENGTH_SHORT).show()
            R.id.miReel -> Toast.makeText(this,"Selected Reel",Toast.LENGTH_SHORT).show()
            R.id.miLive -> Toast.makeText(this,"Selected Live",Toast.LENGTH_SHORT).show()
        }
        return super.onOptionsItemSelected(item)
    }
}